### PR TITLE
TF-524 Apply Quick Search as New Design - Mobile

### PR DESCRIPTION
**Pull request by johnnguyenit39**

### CODE REVIEW

**Comment by hoangdat** [comment]
- build failed
- conflict file

File: lib/features/thread/presentation/thread_view.dart [hunk 1]
```diff
@@ -44,15 +44,15 @@ class ThreadView extends GetWidget<ThreadController> with AppLoaderMixin {
...
                       : const SizedBox.shrink()),
                     _buildSearchInputFormForMobile(context),
-                    Container(
-                        color: BuildUtils.isWeb ? AppColor.colorBgDesktop : Colors.white,
-                        padding: EdgeInsets.zero,
-                        child: _buildSearchButtonViewForMobile(context)),
+                    // Container(
+                    //     color: BuildUtils.isWeb ? AppColor.colorBgDesktop : Colors.white,
+                    //     padding: EdgeInsets.zero,
+                    //     child: _buildSearchButtonViewForMobile(context)),
```
Comment (hoangdat) [inline lib/features/thread/presentation/thread_view.dart:55]: what is it?

File: lib/features/thread/presentation/thread_view.dart [hunk 2]
```diff
@@ -119,49 +119,54 @@ class ThreadView extends GetWidget<ThreadController> with AppLoaderMixin {
...
-          child: (SearchBarView(_imagePaths)
-              ..hintTextSearch(BuildUtils.isWeb ? AppLocalizations.of(context).search_emails : AppLocalizations.of(context).hint_search_emails)
-              ..addOnOpenSearchViewAction(() => controller.enableSearch(context)))
-            .build());
-      } else {
-        return const SizedBox.shrink();
-      }
-    });
-  }
+  // Widget _buildSearchButtonViewForMobile(BuildContext context) {
```
Comment (hoangdat) [inline lib/features/thread/presentation/thread_view.dart:122]: idem

File: lib/features/thread/presentation/thread_view.dart [hunk 3]
```diff
@@ -119,49 +119,54 @@ class ThreadView extends GetWidget<ThreadController> with AppLoaderMixin {
...
+        );
       } else {
         return const SizedBox.shrink();
       }
     });
   }
 
   Widget _buildSearchForm(BuildContext context) {
-    return (SearchAppBarWidget(
+     return (QuickSearchAppBarWidget(
```
Comment (hoangdat) [inline lib/features/thread/presentation/thread_view.dart:155]: what is the different from `SearchAppBarWidget`?

File: lib/l10n/intl_messages.arb [hunk 4]
```diff
@@ -1,1153 +0,0 @@
-{
```
Comment (hoangdat) [inline lib/l10n/intl_messages.arb:1]: Why we need to delete this file?

**Comment by hoangdat** [comment]
We still miss:
- Store recent search in hive database.
- Drop down list recent search.
- Limit the recent search to 10.

I can not find the logic related to filter search result by selected mailbox also.

**Comment by hoangdat** [comment]
@johnnguyenit39  please don't `force` push. It's hard for the reviewer to follow your changes

File: lib/l10n/intl_messages.arb [hunk 5]
```diff
@@ -1,3 +1,4 @@
+
```
Comment (hoangdat) [inline lib/l10n/intl_messages.arb:1]: what is it?

**Review by dab246 — COMMENTED** [review]
- Please attach the demo video on PR for easy tracking.
- You should pay attention to the responsiveness of tablet and browser screens when adding this quick search. Goodluck!

**Pull request closed without merging**